src/Web/Telegram/API/Bot/API.hs

```diff
@@ -68,182 +73,179 @@ newtype Token = Token Text
 telegramBaseUrl :: BaseUrl
 telegramBaseUrl = BaseUrl Https "api.telegram.org" 443 ""
 
--- | Type for token
-type TelegramToken = Capture ":token" Token
-
 -- | Telegram Bot API
 type TelegramBotAPI =
```

Comment:
In respect to token issue, what is you thought on doing it as nested api as descibed here? 
http://haskell-servant.readthedocs.io/en/stable/tutorial/Server.html#nested-apis